test/client/router.core.spec.js

```diff
@@ -633,7 +637,7 @@ Tinytest.add('Client - Router - base path - url generation', (test) => {
 
 function setBasePath(path) {
   FlowRouter._initialized = false;
-  FlowRouter._basePath = path;
+  FlowRouter.basePath(path);
```

Comment:
@jankapunkt  Wouldn’t it be major changes?
Maybe we can change `_basePath` to setter instead of function?